packages/db/src/schema.prisma

```diff
@@ -40,6 +40,8 @@ model AuthUser {
   languageRoles      LanguageMemberRole[]
   /// The list of emails awaiting verification.
   emailVerifications UserEmailVerification[]
+  /// The list of gloss updates this user has performed.
+  glossHistory       GlossHistory[]
```

Comment:
@arrocke I'm going to bring up one concern with your original design....

Should we really name the model `GlossHistory`? It seems like it would be much more accurate to name it `GlossHistoryEntry`. Then `glossHistory` would be of type `GlossHistoryEntry[]`, which would make much more sense.

Reply:
If I were designing it, I would probably name it `GlossUpdate` over `GlossHistoryEntry`

Reply:
I just don't think it makes sense for the individual items to be of type `GlossHistory`. That could cause us confusion in the future. You would expect something called "history" to be an array type, not a scalar type.

Reply:
I'm fine calling it `GlossHistoryEntry`. That makes sense to me

Reply:
Done.